src/controllers/router.js

```diff
@@ -6,25 +6,47 @@ const oauthCode = require('./OAuth/code.js');
 const oauthToken = require('./OAuth/token.js');
 const checkLoggedIn = require('../middleware/checkLoggedIn.js');
 
+router.get('*', (req, res, next) => {
+  console.log(req.url);
```

Comment:
use the npm module morgan. Put it in `dev` mode so it does not interfere with heroku's logging